<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Leonardo Alves da Costa</author>
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PortugolIDE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PortugolIDE.qrc">
    <normaloff>:/images/PortugolIDE.png</normaloff>:/images/PortugolIDE.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSaveAll"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>&amp;Debug</string>
    </property>
    <addaction name="actionStartDebugging"/>
    <addaction name="actionStopDebugger"/>
    <addaction name="separator"/>
    <addaction name="actionStepOver"/>
    <addaction name="actionStepInto"/>
    <addaction name="actionStepOut"/>
    <addaction name="separator"/>
    <addaction name="actionToggleBreakpoint"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionCascade"/>
    <addaction name="actionTileHorizontal"/>
    <addaction name="actionTile_Vertical"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBuild"/>
   <addaction name="menuDebug"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/file.png</normaloff>:/images/file.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/Folder.png</normaloff>:/images/Folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/disk.png</normaloff>:/images/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/disk.png</normaloff>:/images/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/disk_multiple.png</normaloff>:/images/disk_multiple.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/close_window.png</normaloff>:/images/close_window.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/close_all_windows.png</normaloff>:/images/close_all_windows.png</iconset>
   </property>
   <property name="text">
    <string>Close A&amp;ll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/printer.png</normaloff>:/images/printer.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="toolTip">
    <string>Print the current document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/door_in.png</normaloff>:/images/door_in.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/cog.png</normaloff>:/images/cog.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Compile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/resultset_next.png</normaloff>:/images/resultset_next.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStartDebugging">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/bug.png</normaloff>:/images/bug.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start Debugging</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStopDebugger">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;top Debugger</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <action name="actionStepOver">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/step_over.png</normaloff>:/images/step_over.png</iconset>
   </property>
   <property name="text">
    <string>Step &amp;Over</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionStepInto">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/step_in.png</normaloff>:/images/step_in.png</iconset>
   </property>
   <property name="text">
    <string>Step &amp;Into</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionStepOut">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/step_out.png</normaloff>:/images/step_out.png</iconset>
   </property>
   <property name="text">
    <string>Step O&amp;ut</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionToggleBreakpoint">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/tag_blue.png</normaloff>:/images/tag_blue.png</iconset>
   </property>
   <property name="text">
    <string>Toggle &amp;Breakpoint</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/page_copy.png</normaloff>:/images/page_copy.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/page_paste.png</normaloff>:/images/page_paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>&amp;Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionTileHorizontal">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/application_tile_horizontal.png</normaloff>:/images/application_tile_horizontal.png</iconset>
   </property>
   <property name="text">
    <string>Tile &amp;Horizontal</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/application_cascade.png</normaloff>:/images/application_cascade.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cascade</string>
   </property>
  </action>
  <action name="actionTile_Vertical">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/application_tile_vertical.png</normaloff>:/images/application_tile_vertical.png</iconset>
   </property>
   <property name="text">
    <string>Tile &amp;Vertical</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>&amp;Next</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>&amp;Previous</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="PortugolIDE.qrc">
     <normaloff>:/images/options.png</normaloff>:/images/options.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PortugolIDE.qrc"/>
 </resources>
 <connections/>
</ui>
